src/components/modules/blockManager.ts

```diff
@@ -649,11 +680,107 @@ export default class BlockManager extends Module {
   private bindEvents(block: Block): void {
```

Comment:
better to change name for `bindEventToBlock` because there is a `enableModuleEvents ` with the same meaning